app/assets/stylesheets/global/_helper.scss

```diff
@@ -2,30 +2,53 @@
 // helper classes
 //--------------------------------------------------------------------------------
 // - backgrounds
+// - display
 // - text
 //--------------------------------------------------------------------------------
-
-
-//--------------------------------------------------------------------------------
 // backgrounds
 //--------------------------------------------------------------------------------
-.bg__grey {
```

Comment:
Yea I guess, I don't think the file will get big enough in this project to split it out though?